### DISCUSSION TITLE
"Last seen" info in user profile doesn't match with osu!web

### DISCUSSION THREAD

**Discussion by Semro** [ppy/osu#16157] [Q&A] [answered]
As you can see in the screenshots below, for osu!lazer it's 6 month ago, for osu!web it's 7 month ago.
osu!lazer:
![image](https://user-images.githubusercontent.com/3332159/146672399-df135273-ed46-4814-ae44-173ad588b35b.png)
osu!web:
![image](https://user-images.githubusercontent.com/3332159/146672444-c34aaf7a-d1f3-4ab6-ba6e-a4826dc6c263.png)
osu!lazer version: 2021.1218.0

**Comment by bdach** [comment, marked as answer]
this will be due to differences in rounding between libraries used in web ([timeago](https://timeago.org/)) and client ([humanizer](https://github.com/Humanizr/Humanizer/tree/main/src)). very low priority item to fix.
